pandas/tests/tslibs/test_api.py

```diff
@@ -22,6 +22,7 @@ def test_namespace():
                   'timezones']
 
     api = ['NaT',
+           'NaTType',
```

Comment:
This test was failing because of `NaTType` addition to API so simply added here @jbrockmendel